apollo-common/src/main/java/com/ctrip/framework/apollo/common/utils/ConfigValidator.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.common.utils;
+
+import com.ctrip.framework.apollo.core.utils.StringUtils;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.util.Assert;
+
+/**
+ * Config Validator
+ *
+ * @author Lv Lifeng
+ */
+public class ConfigValidator {
+
+  private final ConfigurableApplicationContext applicationContext;
+
+  public ConfigValidator(ConfigurableApplicationContext applicationContext) {
+    this.applicationContext = applicationContext;
+    checkDataBaseName();
+  }
+
+  private void checkDataBaseName() {
+    String dataSourceUrl = System.getProperty("spring.datasource.url");
+    if (!StringUtils.isBlank(dataSourceUrl)
+        && dataSourceUrl.contains("/")
+        && dataSourceUrl .contains("?")
```

Comment:
Is `?` mandatory?

Reply:
Yes, this is how it is handled now. If there is a better way, it can be replaced.

Reply:
I think we need to handle case where there is no `?`

Reply:
I didn't think about it before,I will add this logic.:-)